docs/flashbots-core/searchers/coinbase-payments.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Using coinbase.transfer()
```

Comment:
i would consider changing the file_name to be the doc_title to reduce complexity